src/components/SettlementButton.tsx

```diff
@@ -184,7 +182,9 @@ function SettlementButton({
         if (isExpenseReport && shouldShowPaywithExpensifyOption) {
             buttonOptions.push(paymentMethods[CONST.IOU.PAYMENT_TYPE.VBBA]);
         }
-        buttonOptions.push(paymentMethods[CONST.IOU.PAYMENT_TYPE.ELSEWHERE]);
+        if (isExpenseReport && policy?.reimbursementChoice === CONST.POLICY.REIMBURSEMENT_CHOICES.REIMBURSEMENT_MANUAL) {
```

Comment:
@luacmartins i made a 1:1 request. But I'm not able to see the pay option.
I can see it on staging, but not on this PR.

Staging: 
<img width="501" alt="image" src="https://github.com/Expensify/App/assets/29673073/0e8b8376-84ff-4f12-9029-f88419da882f">
<img width="1032" alt="image" src="https://github.com/Expensify/App/assets/29673073/5ff1e36d-93f5-4fc7-bc34-18798a84e06d">


Dev:
<img width="1135" alt="image" src="https://github.com/Expensify/App/assets/29673073/b6ccea04-9694-4734-ac9b-f94d37d15075">

<img width="522" alt="image" src="https://github.com/Expensify/App/assets/29673073/9030948f-ee6a-46b4-8d92-1693bfeec1ff">

Reply:
Nice catch! That would have been really bad 😬 I pushed a fix!